features/support/user.rb

```diff
@@ -2,6 +2,11 @@ module UserHelpers
   def find_or_create_new_user(login, password, activate: true)
     user = User.find_by(login: login)
     if user.blank?
+      # If we're logged in as an admin when we try to use a step that invokes
+      # this method to create a user, user creation will fail because only some
+      # admins can edit certain fields on pseuds. Setting current_user to nil
+      # will bypass that.
+      User.current_user = nil
```

Comment:
Is this not going to clobber the existing logged-in-as-admin situation for the scenario or step that invoked the user creation?

(Alternatively, do we need to add these guardrails to all user-creating support methods? If they're not used now, might we encounter this in the future?)

Reply:
I was worried about that, too, but it doesn't seem to. For example, these tests still pass:

https://github.com/otwcode/otwarchive/blob/d99747574ed348866cdeed6cda507b7636612582/features/other_a/invite_request.feature#L125-L146

I'll add it to `ensure_user` as well, since if I swap the order of the steps in this scenario (just to pick a random scenario), I get the same error.

https://github.com/otwcode/otwarchive/blob/d99747574ed348866cdeed6cda507b7636612582/features/admins/users/admin_fnok.feature#L35-L41